recipes/delete-me/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "giotto-learn" %}
+{% set version = "0.1a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/matteocao/{{ name }}.git
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script:
+    - git submodule update --init --recursive
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - gcc  # [not win]
+    - vs2017  # [win]
+    - cmake
+    - boost >=1.56.*
+    - git
+    - setuptools
+  host:
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.17.0
+    - scipy
+    - scikit-learn >=0.21.3
+  run:
+    - python
+    - numpy >=1.17.0
```

Comment:
Are you compiling against ``numpy``?
if so, please see
https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=compiler%20cxx#building-against-numpy